assets/out

```diff
@@ -73,6 +73,7 @@ load=$(jq -r '.params.load // ""' < $payload)
 load_base=$(jq -r '.params.load_base // ""' < $payload)
 build=$(jq -r '.params.build // ""' < $payload)
 cache=$(jq -r '.params.cache' < $payload)
+cache_tag=$(jq -r '.params.cache_tag // "${tag_name}"' < $payload)
```

Comment:
This does not work as intended. ${tag_name} is not expanded inside single quotes.